### PR TITLE
Convert quotas to numeric values

### PR DESCRIPTION
**Pull request by zakiva**
Parse the quotas as numeric (int or float) values instead of string. 

Depends on https://github.com/ManageIQ/manageiq-schema/pull/44

@simon3z @zeari Please review

### CODE REVIEW

**Comment by cben** [comment]
@zeari how does this relate to your suffixed->value conversion quest?

**Comment by zeari** [comment]
> @zeari how does this relate to your suffixed->value conversion quest?

Its about the same and relies on mostly the same types of conversions

**Comment by zakiva** [comment]
@moolitayer Please review

File: app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb [hunk 1]
```diff
@@ -869,28 +869,40 @@ def parse_resource_quota(resource_quota)
...
 
       resource_quota.status.used.to_h.each do |resource_name, quota|
-        new_result_h[resource_name][:quota_observed] = quota
+        add_numeric_quota(new_result_h[resource_name], quota, "quota_observed")
       end
 
       new_result_h.values
     end
 
+    def add_numeric_quota(h, quota, key)
```
Comment (moolitayer) [inline app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb]: `s/key/prefix/`

**Comment by cben** [comment]
@zakiva tests in #22 might help (you probably knew this)

File: app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb (hunk 1 above)
Comment (zakiva) [inline app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb]: done

**Comment by zakiva** [comment]
@moolitayer `parse_quantity` method was added through #22 by @zeari with some tests (removed it from here after rebase). I updated the quotas specs and added some other example values.

**Comment by moolitayer** [comment]
@zeari @cben review pls

**Comment by moolitayer** [comment]
The code parsing quotas[1] is a little hard to read already and getting harder (I think I wrote it ¯\_(ツ)_/¯). 

@zakiva maybe you can add in another PR an example for the value of resource_quota.spec.<x>.to_h?

I'm sure you'll find the best way to do it

[1]
```
resource_quota.X.Y.to_h.each do |resource_name, quota|
  new_result_h[resource_name][:quota_desired] = quota
end
```

**Comment by moolitayer** [comment]
@zakiva can you check Travis?

**Comment by zakiva** [comment]
Updated the PR according to the schema changes, we'll probably store all values as floats (makes this PR much simpler).

**Comment by Fryguy** [comment]
Merged the schema, so bouncing this PR.

**Comment by zakiva** [comment]
@moolitayer this should be ready now

**Pull request merged**